codalab/client/local_bundle_client.py

```diff
@@ -804,15 +803,23 @@ def new_worksheet(self, name, ensure_exists):
             'title': None,
             'frozen': None,
             'items': [],
-            'owner_id': self._current_user_id()
+            'owner_id': '0' if spec_util.is_dashboard(name) else self._current_user_id()
         })
         self.model.new_worksheet(worksheet)
 
         # Make worksheet publicly readable by default
         self.set_worksheet_perm(worksheet.uuid, self.model.public_group_uuid, 'read')
-
+        if spec_util.is_dashboard(name):
+            self.populate_dashboard(worksheet)
         return worksheet.uuid
 
+    def populate_dashboard(self, worksheet):
+        lines = [line.rstrip() for line in open('../objects/dashboard.ws').readlines()]
```

Comment:
I don't think this will access the right file in general.  'cl' could be run from anywhere...

One compromise is to have a single python file dashboard_lines.py which contains 
DASHBOARD_LINES = """
...
"""
and just load that.